src/Command/WpCliCommandTrait.php

```diff
@@ -8,7 +8,8 @@
 /**
  * Utility trait for ensuring singleton instances of WP CLI commands.
  *
- * If your command class uses static functions for commands – you probably don't need this trait.
+ * If your command class uses static functions for commands – don't use this trait as it will cause a fatal error
```

Comment:
I think _If your command class uses static functions for commands – you shouldn't use this trait._

Is better here